pkg/workflows/wasm/host/module.go

```diff
@@ -366,22 +386,27 @@ func (m *Module) Run(request *wasmpb.Request) (*wasmpb.Response, error) {
 	_, err = start.Call(store)
 	switch {
 	case containsCode(err, wasm.CodeSuccess):
-		resp, innerErr := m.r.get(request.Id)
+		storedRequest, innerErr := m.requestStore.get(request.Id)
```

Comment:
We should actually pop the item here, not just get it. This is so that we ensure we're removing items once we've completed the request.